lib/features/email/presentation/widgets/message_content_tile_builder.dart

```diff
@@ -0,0 +1,95 @@
+
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';
+import 'package:photo_view/photo_view.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/message_content.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/text_format.dart';
+
+typedef OnOpenExternalUrlActionClick = void Function(String url);
+
+class MessageContentTileBuilder {
+
+  final MessageContent _messageContent;
+  final ImagePaths _imagePaths;
+
+  OnOpenExternalUrlActionClick? _onOpenExternalUrlActionClick;
+
+  MessageContentTileBuilder(this._messageContent, this._imagePaths);
+
+  MessageContentTileBuilder onOpenExternalUrlActionClick(OnOpenExternalUrlActionClick onOpenExternalUrlActionClick) {
+    _onOpenExternalUrlActionClick = onOpenExternalUrlActionClick;
+    return this;
+  }
+
+  Widget build() {
+    return Theme(
+      data: ThemeData(
+        splashColor: Colors.transparent,
+        highlightColor: Colors.transparent),
+      child: MediaQuery(
+        data: MediaQueryData(padding: EdgeInsets.zero),
+        child: Center(
+          child: _messageContent.textFormat == TextFormat.PLAIN
+            ? Text('${_messageContent.content}', style: TextStyle(fontSize: 12, color: AppColor.mailboxTextColor))
+            : HtmlWidget(
```

Comment:
Please set up `webViewJs = false`, we don't want to run any script here.

Reply:
beside that. IMO, this library quite cool. Please try with it.

Reply:
but the parsing time is quite low, we need to make a comparison with other library? WDYT? @chibenwa @dab246

Reply:
What do you base that claim on?